modules/core/client/directives/pagination.client.directive.html

```diff
@@ -0,0 +1,7 @@
+<div class="list-group">
+    <div data-ng-repeat="item in pagedItems">
```

Comment:
+1 on using `track by` 

@ilanbiala I think this is a good idea... using the `limitTo`, in conjunction with the filters and `track by` will enhance the performance. User lists tend to get unwieldy. 

@trainerbill You can use the pagination buttons to mimic a 'scroll' by updating the filter range & offsets based on how the user is interacting with the pagination.